packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.test.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react'
+import 'react-native'
+import { fireEvent, render, waitForElement } from 'react-native-testing-library'
+import { Provider } from 'react-redux'
+import { CELO_REWARDS_LINK } from 'src/config'
+import ConsumerIncentivesHomeScreen from 'src/consumerIncentives/ConsumerIncentivesHomeScreen'
+import { fetchConsumerRewardsContent } from 'src/consumerIncentives/contentFetcher'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+
+const mockScreenProps = getMockStackScreenProps(Screens.ConsumerIncentivesHomeScreen)
+
+const createStore = (numberVerified: boolean) =>
+  createMockStore({
+    app: { numberVerified },
+  })
+
+jest.mock('src/consumerIncentives/contentFetcher', () => ({
+  fetchConsumerRewardsContent: jest.fn(() =>
+    Promise.resolve({
+      en: {
+        title: 'Title',
+        description: 'Description',
+        subtitle1: 'Subtitle 1',
+        body1: 'Body 1',
+        subtitle2: 'Subtitle 2',
+        body2: 'Body 2',
```

Comment:
Looks simple and does what we need! 👍 
Do you see this needing richer content capabilities (links, images)?

Reply:
Maybe in the future, yes. I tried to think a bit about it but didn't come up with anything super robust and since we don't need it right now I let it be for now.

For links, a possibility would be to have an array of links and then have the part of the text we want to use it be wrapped by <N>link text</N>, N being the index on the link array. 

Didn't think of hte images part too much, but there's probably a way to add that too.